src/FuseProcess.ts

```diff
@@ -22,6 +23,40 @@ export class FuseProcess {
         return this;
     }
 
+		require(close?: ((exports: any) => void)|(() => void)) {
+			function getMainExport(mdl) {
+				return mdl && mdl.FuseBox && mdl.FuseBox.mainFile ?
+						mdl.FuseBox.import(mdl.FuseBox.mainFile) :
```

Comment:
Well, the exports of the module are used in 2 cases :
1- as the value of resolution. This one is optional but the exported values are given to the script in fuse - this could be useful for someone, somebody who write a `close` function in the fuse file for example
2- to find back the `close` functions indeed, in order for instance to stop express to be able to re-launch it afterward